### ISSUE TITLE
Define dispatch and costs for Germany and neighbours

### ISSUE THREAD

**Issue by lukasol** [openego/eTraGo#24]
Currently, dispatch is presented as a total system with all neighbouring countries being one node. 

- [x] differentiate between German and non-German load and dispatch
- [x] present the difference as import or export
- [x] goal: get one `network` that only presents German load and dispatch and all import and export separately
- [x] adjust the plotting functions, i.e. stacked_gen accordingly

**Comment by lukasol** [comment]
is it realistic to finish this by release date tomorrow?

**Comment by wolfbunke** [comment]
no.

**Comment by MarlonSchlemminger** [comment]
Marius, did you already start working on this?

**Comment by ulfmueller** [comment]
@mariusves please answer the previous question. if the answer is no, @MarlonSchlemminger  will work on this.

by the way:
@MarlonSchlemminger with the @ the person you are asking gets a notification...

**Comment by mariusves** [comment]
@marlonschlemminger no, I haven't started working on this. Go ahead! 
Thanks @ulfmueller for the mention, I somehow didn't receive a notification beforehand even though I'm subscribed to this thread....

**Comment by MarlonSchlemminger** [comment]
@ulfmueller 
Added new branch features/german-dispatch to both etrago and egopowerflow. Added function clip_foreign in utilities.py which takes a network and returns a new network where all foreign components are deleted. So far I hardcoded the coordinates of the foreign buses to cut them out, any other ideas on that? The difference between the remaining german load and generation is stored as a new component network.foreign_trade, where positive values are imports and negative values are exports. 

Adjusted plot_stacked_gen to plot the imports as well (see figure). Exports would be the generation above the dotted line.
![stacked_gen_imports](https://user-images.githubusercontent.com/22960010/31074538-6b66d20e-a771-11e7-81cf-68060eda83f7.png)
